values.yaml

```diff
@@ -90,34 +90,56 @@ injector:
   # Configures all Vault Agent sidecars to revoke their token when shutting down
   revokeOnShutdown: false
 
-  # namespaceSelector is the selector for restricting the webhook to only
-  # specific namespaces.
-  # See https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-namespaceselector
-  # for more details.
-  # Example:
-  # namespaceSelector:
-  #    matchLabels:
-  #      sidecar-injector: enabled
-  namespaceSelector: {}
-  # objectSelector is the selector for restricting the webhook to only
-  # specific labels.
-  # See https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-objectselector
-  # for more details.
-  # Example:
-  # objectSelector:
-  #    matchLabels:
-  #      vault-sidecar-injector: enabled
-  objectSelector: {}
-
-  # Configures failurePolicy of the webhook. The "unspecified" default behaviour deoends on the
-  # API Version of the WebHook.
-  # To block pod creation while webhook is unavailable, set the policy to `Fail` below.
-  # See https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#failure-policy
-  #
-  failurePolicy: Ignore
-
-  # Extra annotations to attach to the webhook
-  webhookAnnotations: {}
```

Comment:
While I like the grouping under `webhook`, we try to maintain backwards compatibility for previously written values files. Please could we keep these settings in place? Up to you whether you'd like to duplicate options between here and `webhook`, and comment these ones as deprecated, or just use a flat structure.

Reply:
Good point! I will go ahead and duplicate the options and deprecate the old ones.

Reply:
I have gone ahead and edited the values.yaml, values.schema.json and the webhook deployment file. Next commit will contain the small unit test and the fix for the _helpers.tpl